doc/api/cli.md

```diff
@@ -984,6 +984,34 @@ surface on other platforms, but the performance impact may be severe.
 This flag is inherited from V8 and is subject to change upstream. It may
 disappear in a non-semver-major release.
 
+### `--env-file=config`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Loads environment variables from a file relative to the current directory,
+making them available to applications on `process.env`. The [environment
+variables which configure Node.js][environment_variables], such as `NODE_OPTIONS`,
+are parsed and applied. If the same variable is defined in the environment and
+in the file, the value from the environment takes precedence.
+
+The format of the file should be one line per key-value pair of environment
+variable name and value separated by `=`:
+
+```text
+PORT=3000
+```
+
+Any text after a `#` is treated as a comment:
+
+```text
+# This is a comment
+PORT=3000 # This is also a comment
+```
```

Comment:
Somewhere, perhaps in this section, the exact syntax should be specified. In other environments, it is common to use shell syntax for `.env` files, which is why I am somewhat confused to see examples with backticks here, which would have a different meaning in other `.env` files. Similarly, it should be documented how escaping works.

I don't have much context here so I don't know where this syntax is coming, but I think most users would be happy with plain strings and JSON string literals. Anything more complex than that should either match JavaScript string literals or requires justification, I guess.

Reply:
The syntax and all tests are coming from the original `dotenv` package, and they are based on top of [ini file format](https://en.wikipedia.org/wiki/INI_file#Format)

Reply:
> they are based on top of [ini file format](https://en.wikipedia.org/wiki/INI_file#Format)

I think this would be good to document, and a link to the Wikipedia article like you have here seems appropriate.

Reply:
The reference to the ini file format doesn't seem to match the syntax here. One of the main features of ini files are sections, which don't exist here as far as I can tell. Backticks are also not mentioned on the Wikipedia page.

Perhaps, if we do add `.env` files to node, we could consider better file formats than something with no clear specification that is loosely based on the ancient ini file format. At least we don't have to worry about breaking things for now.

Reply:
There isn't any spec for the `.env` file format. I'll just remove it, and we can follow up on a different pull request for this.